Frameworks/GLKit/GLKEffect.mm

```diff
@@ -87,7 +87,7 @@ - (BOOL)prepareShaders {
 }
 
 - (void)prepareToDraw {
-    BOOL success = FALSE;
+    bool success = false;
```

Comment:
Fair enough.  My comment on BOOL vs bool specifically applies to the params and return values in public APIs.